packages/cfg-core/lib/Pair.ts

```diff
@@ -1,3 +1,20 @@
+/*
+ * Copyright 2020-2023 Delft University of Technology and SynTest contributors
+ *
+ * This file is part of SynTest Framework - SynTest Core.
```

Comment:
Inconsistent with the other files. Although this package is within the SynTest Core repo it is not part of the SynTest Core package. We could just state it is part of the SynTest Framework.

```suggestion
 * This file is part of SynTest Framework.
```
We should do this for the other files in this package as well.